packages/_debian-common/changelog

```diff
@@ -6,6 +6,7 @@ libsuperiormysqlpp (0.4.1) UNRELEASED; urgency=medium
 
   [ Michal Merc ]
   * feat: Implement psQuery/psParamQuery/psResultQuery convenience functions
```

Comment:
Please increase MINOR version, introducing new methods and deprecating another one is not what I expect from patch version change